python/sglang/srt/managers/schedule_batch.py

```diff
@@ -109,6 +109,7 @@
     "weight_loader_disable_mmap",
     "enable_triton_kernel_moe",
     "enable_flashinfer_mxfp4_moe",
+    "enable_flashinfer_mxfp4_bf16_moe",
```

Comment:
Can we use a different arg name like `flashinfer_mxfp4_moe_config`? It's more clear and extensible.